spec.html

```diff
@@ -84,32 +84,28 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
       1. Let _remainingElementsCount_ be a new Record { [[Value]]: 1 }.
       1. Let _index_ be 0.
       1. Let _promiseResolve_ be ? Get(_constructor_, `"resolve"`).
-      1. If IsCallable(_promiseResolve_) is *false*, throw a *TypeError* exception.
+      1. If ! IsCallable(_promiseResolve_) is *false*, throw a *TypeError* exception.
       1. Repeat,
         1. Let _next_ be IteratorStep(_iteratorRecord_).
         1. If _next_ is an abrupt completion, set _iteratorRecord_.[[Done]] to *true*.
         1. ReturnIfAbrupt(_next_).
         1. If _next_ is *false*, then
           1. Set _iteratorRecord_.[[Done]] to *true*.
-          1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
```

Comment:
These lines were important, I think (see #27); is there a reason they were removed?